docs/src/pages/Identifier/0-types.vue

```diff
@@ -11,11 +11,38 @@ Include `aria-label` when there is no text equivalent for the identifier. This i
     <FdIdentifier backgroundColor="accent-1" icon="washing-machine" />
     <FdIdentifier circle backgroundColor="accent-1" icon="washing-machine" />
     <FdIdentifier aria-label="Christian Kienle">CK</FdIdentifier>
-    <FdIdentifier thumbnail url="/images/headshot-male.jpg" />
-    <FdIdentifier circle thumbnail url="/images/headshot-male.jpg" />
+    <FdIdentifier
+      thumbnail
+      :url="
+        env === 'production'
+          ? 'fundamental-vue/images/headshot-male.jpg'
+          : '/images/headshot-male.jpg'
+      "
+    />
```

Comment:
This throws a failed prop type error, but I'm not sure how else to do this.

Reply:
You could solve this by replacing

```
<script lang="ts">
export Vue.extend({
  data() {
    return {
      env: process.env.NODE_ENV
    };
  }
});
</script>
```

with 

```
<script>
export default {
  data() {
    return {
      env: process.env.NODE_ENV
    };
  }
};
</script>
```

in `pages/identifier/0-types.vue`.

This is not something you caused and it does not have any impact on the functionality because even though the type is incorrect we handle it correctly.

We have tests that would have prevented exactly the problem or error message you found.  I think we should turn them on. I can do that in a different PR.